packages/qwik/src/core/shared/shared-serialization.ts

```diff
@@ -297,13 +300,17 @@ const inflate = (
       signal.$effects$ = new Set(d.slice(5) as EffectSubscriptions[]);
       break;
     }
+    // Inflating a SerializerSignal is the same as inflating a ComputedSignal
+    case TypeIds.SerializerSignal:
     case TypeIds.ComputedSignal: {
       const computed = target as ComputedSignal<unknown>;
-      const d = data as [QRLInternal<() => {}>, any, unknown?];
+      const d = data as [QRLInternal<() => {}>, EffectSubscriptions[] | null, unknown?];
       computed.$computeQrl$ = d[0];
-      computed.$effects$ = d[1];
-      if (d.length === 3) {
+      computed.$effects$ = new Set(d[1]);
+      if (d.length >= 3) {
```

Comment:
`>=3` seems like magic here. Can we pull this out to new function which has a descriptive name so it is clear what condition are we checking here.